<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <title>Markdown_Resume</title>
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    div.columns{display: flex; gap: min(4vw, 1.5em);}
    div.column{flex: auto; overflow-x: auto;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    /* The extra [class] is a hack that increases specificity enough to
       override a similar rule in reveal.js */
    ul.task-list[class]{list-style: none;}
    ul.task-list li input[type="checkbox"] {
      font-size: inherit;
      width: 0.8em;
      margin: 0 0.8em 0.2em -1.6em;
      vertical-align: middle;
    }
    .display.math{display: block; text-align: center; margin: 0.5rem auto;}
  </style>
  <link rel="stylesheet" href="pandoc.css" />
</head>
<body>
<h1 id="césar-a.-lizárraga">César A. Lizárraga</h1>
<p>email: <a href="mailto:calizarr@gmail.com"
class="email">calizarr@gmail.com</a><br />
github: <a
href="http://github.com/calizarr">http://github.com/calizarr</a><br />
LinkedIn: <a
href="http://www.linkedin.com/in/calizarr">http://linkedin.com/in/calizarr</a></p>
<h2 id="relevant-experience">Relevant Experience</h2>
<p><strong><a href="https://www.cibotechnologies.com/">CiBO
Technologies</a> (St. Louis, MO): July 2017 - Present</strong></p>
<p><strong><em>Senior Software Engineer (Infrastructure)</em></strong> →
<strong>March 2021 - Present</strong></p>
<ul>
<li>The description from Interim Lead of Infrastructure applies except
for the first bullet point.</li>
</ul>
<p><strong><em>Interim Lead of Infrastructure (Software
Enginer)</em></strong> → <strong>April 2020 - March 2021</strong></p>
<ul>
<li>Lead the infrastructure team, planned the near future, and
collaboratively made decisions with the leadership team as a whole.</li>
<li>Support the data science / computer vision team in their research
and development by helping set up and maintain tools that allow for
massive parallelization and ease of use within our k8s clusters
e.g. Ray, MLFlow, etc. and new clusters when necessary</li>
<li>Support general engineering teams in their goals by helping with
CI/CD, deployment configurations for all environments, and general
knowledge involved in our infrastructure as a whole.</li>
<li>Maintain, support, and improve upon our currently existing k8s
clusters and the applications that help them run smoothly including but
not limited to webhooks, Linkerd, Prometheus, Grafana, ArgoCD, Argo
Workflows, etc.</li>
<li>Maintain, support, and improve the cloud infrastructure, currently
AWS, including but not limited to DynamoDB, IAM, S3 Buckets, Route53,
EC2 Instances, etc.</li>
<li>Help architect and design new infrastructure policies, systems, etc.
as well as improve upon or refactor currently existing systems for the
purpose of ease of use, reliability, and redundancy</li>
<li>Coordinate with our team and others across the organization in terms
of security wihin our clusters, public facing domains, cloud
infrastructure, etc. for goals such as SOC2 Compliance, AWS Architecture
reviews, Escrow responsibilities, etc.</li>
</ul>
<p><strong><em>Engagement Engineering Lead (Project Lead / Software
Engineer)</em></strong> → <strong>July 2017 - April 2020</strong></p>
<ul>
<li>Led a small team (4-5 engineers) in custom application development
for a strategic partnership client that resulted in a multimillion
dollar contract</li>
<li>Produced sprint planning and project planning documentation while
working with business development and leadership</li>
<li>Interfaced multiple times per week with various members of the
client team to ensure their needs were met</li>
<li>Reduced client research time from months to days using a high level
business development and product concept to design and architect an
industry leading set of tools</li>
<li>Developed several services, APIs, and API clients in Scala and
Python from prototype to deployment</li>
</ul>
<p><strong><a href="http://www.nanaya.co">Nanaya</a> (St. Louis, MO):
July 2014 - Present</strong></p>
<p><strong><em>Co-founder / Infrastructure &amp; Sofware Engineer /
Statistician</em></strong></p>
<ul>
<li>Supported, managed, and updated infrastructure for application
averaging 50 users per day and about 300,000 total</li>
<li>Migrated original application infrastructure to a container based
model</li>
<li>Facilitated communication between developers and R&amp;D Team
members</li>
<li>Managed team member contributions and source code version
control</li>
<li>Developed, verified, and tested algorithm(s) for application</li>
</ul>
<p><strong><a href="https://www.danforthcenter.org/">Donald Danforth
Plant Science Center</a> (St. Louis, MO)</strong></p>
<p><strong><em>Senior Computational Scientist</em></strong> →
<strong>October 2016 - July 2017</strong></p>
<ul>
<li>Provided computational interface for research groups</li>
<li>Trained lab technicians, research scientists, and graduate students
in computational infrastructure use</li>
<li>PheNode: Developed prototype of Arduino &amp; Raspberry Pi
controlled field canopy sensor system which became <a
href="https://www.agrelaeco.com/">AgrelaEco</a>
<ul>
<li><a
href="https://github.com/calizarr/EPSCoR_Bramble_GH9C">PhenoPiSight: A
Fixed Camera Greenhouse-based Phenotyping platform</a></li>
</ul></li>
</ul>
<p><strong><em>LabTech (Bioinformatics/Statistics) Mockler
Lab</em></strong> → <strong>December 2014 - October 2016</strong></p>
<ul>
<li>Developed, documented, maintained, and tested pipelines for
analyzing high throughput sequencing and imaging data</li>
<li>Analytics, data management, and development for the <a
href="http://genomicscience.energy.gov/research/DOEUSDA/abstracts/2014mockler_abstract.shtml">Brachypodium
ENCODE Project</a> and the <a href="https://missouriepscor.org/">EPSCoR
Project</a></li>
<li>Coordinated with Bioinformatics Core Director for computational
resource management</li>
<li>Performed data analysis on a routine basis</li>
</ul>
<h2 id="projects">Projects</h2>
<h3 id="alvoiarko-homelab">Alvoiarko Homelab</h3>
<ul>
<li>3-4 node Kubernetes cluster
<ul>
<li><a href="https://www.talos.dev/">Talos Images</a>, <a
href="https://cilium.io/">Cilium eBPF Networking</a>, <a
href="https://www.proxmox.com/en/">Proxmox</a>, Bare-metal nodes</li>
<li>Managed by <a href="https://opentofu.org/">OpenTofu</a> with CI/CD
provided by <a
href="https://argo-cd.readthedocs.io/en/stable/">ArgoCD</a> and <a
href="https://tekton.dev/">Tekton</a> with support from <a
href="https://argoproj.github.io/workflows/">Argo Workflows</a></li>
</ul></li>
<li>DHCP and DNS managed by <a href="https://pi-hole.net/">PiHole</a>
and <a
href="https://nlnetlabs.nl/projects/unbound/about/">Unbound</a></li>
<li>Hosting a variety of projects such as
<ul>
<li><a href="https://reconya.com/">Reconya</a>,<a
href="https://github.com/C4illin/ConvertX">ConvertX</a>, <a
href="https://gethomepage.dev/">Homepage Dashboard</a>, <a
href="https://www.plex.tv/">Plex</a>, and more</li>
</ul></li>
</ul>
<h3 id="public-projects">Public Projects</h3>
<h4
id="phenopisight-fixed-camera-greenhouse-based-phenotyping-platform">PhenoPiSight:
Fixed Camera Greenhouse-based Phenotyping
Platform<a name="phenopisight"></a></h4>
<ul>
<li>Used Ansible to automate image capture and transfer of images from
180 Raspberry Pis on a gantry above the greenhouse</li>
<li>Developed pipeline to take captured images and make dense 3D
pointcloud reconstructions (+/- 0.5cm accuracy)</li>
<li>Trained lab technicians to find phenotypes in 3D reconstructions and
compare to ground-based greenhouse measurements</li>
<li><a
href="https://traitcapture.org/pointclouds/by-id/586a428ef7f5667846b1f8a0">Example
of the 3D reconstruction:
https://traitcapture.org/pointclouds/by-id/586a428ef7f5667846b1f8a0</a></li>
</ul>
<h2 id="education">Education</h2>
<p><strong>B.A. Mathematics, 2008</strong><br />
<em>Probability and Statistics</em><br />
Department of Mathematics, Washington University in St. Louis</p>
<h2 id="engineering-skills">Engineering Skills</h2>
<ul>
<li>In order of familiarity:
<ul>
<li>Languages: <em>Scala, Python, R, SQL, Bash, Java</em></li>
<li>Frameworks: <em>Akka, Django</em></li>
<li>Infrastructure &amp; DevOps: <em>Kubernetes, Argo Workflow Engine,
Helm, Ansible, HTCondor</em></li>
<li>Cloud computing: <em>AWS, DigitalOcean</em></li>
<li>Software: <em>GNU Emacs, Git, PostgreSQL, IntelliJ IDEA, NGINX,
Travis CI, Jenkins, RabbitMQ</em></li>
</ul></li>
</ul>
<h2 id="languages">Languages</h2>
<ul>
<li>Fluent: <em>English, Spanish</em></li>
<li>Intermediate: <em>Italian</em></li>
</ul>
<h2 id="publications">Publications</h2>
<ul>
<li><p>Joseph Ballenger, Samuel Kenney, Jared Gordon, Leonardo Chavez,
Joseph G. Duenwald, Katherine M. Murphy, Leonardo W. Lima, <strong>Cesar
Lizarraga</strong>, Malia A. Gehan. <strong>Raspberry Pi–powered
temperature monitoring of growth chamber microclimates</strong>.
https://doi.org/10.1002/aps3.70022 (2025)</p></li>
<li><p>Agnew Erica , Ziegler Greg , Lee Scott , <strong>Lizárraga
César</strong> , Fahlgren Noah , Baxter Ivan , Mockler Todd C. , Shakoor
Nadia. <strong>Longitudinal genome-wide association study reveals early
QTL that predict biomass accumulation under cold stress in
sorghum</strong>. <a
href="https://www.frontiersin.org/journals/plant-science/articles/10.3389/fpls.2024.1278802"><em>Frontiers
in Plant Science Volume 15 - 2024</em></a> DOI=10.3389/fpls.2024.1278802
ISSN=1664-462X</p></li>
<li><p>Nadia Shakoor, Erica Agnew, Greg Ziegler, Scott Lee,
<strong>César Lizárraga</strong>, Noah Fahlgren, Ivan Baxter, Todd C.
Mockler. <strong>Genomewide association study reveals transient loci
underlying the genetic architecture of biomass accumulation under cold
stress in Sorghum</strong>. bioRxiv 760025; doi:
https://doi.org/10.1101/760025 (2019)</p></li>
<li><p>Erica Agnew, Adam Bray, Eric Floro, Nate Ellis, John Gierer,
<strong>César Lizárraga</strong>, Darren O’Brien, Madeline Wiechert,
Todd C. Mockler, Nadia Shakoor, Christopher N. Topp. <strong>Whole‐Plant
Manual and Image‐Based Phenotyping in Controlled Environments</strong>.
<em>Plant Biology 2(1):1-21. https://doi.org/10.1002/cppb.20044
(2017)</em></p></li>
<li><p>He Huang, Malia A. Gehan, Sarah E. Huss, Sophie Alvarez,
<strong>Cesar Lizarraga</strong>, Ellen L. Gruebbling, John Gierer,
Michael J. Naldrett, Rebecca K. Bindbeutel, Bradley S. Evans, Todd C.
Mockler, Dmitri A. Nusinow. <strong>Cross‐species complementation
reveals conserved functions for EARLY FLOWERING 3 between monocots and
dicots</strong>. <em>Plant Direct 1:4 https://doi.org/10.1002/pld3.18
(2017)</em></p></li>
<li><p>Gehan MA, Fahlgren N, Abbasi A, Berry JC, Callen ST, Chavez L,
Doust AN, Feldman MJ, Gilbert KB, Hodge JG, Hoyer JS, Lin A, Liu S,
<strong>Lizárraga C</strong>, Lorence A, Miller M, Platon E, Tessman M,
Sax T. <strong>PlantCV v2: Image analysis software for high-throughput
plant phenotyping</strong>. PeerJ. 2017 Dec 1;5:e4088. doi:
10.7717/peerj.4088. PMID: 29209576; PMCID: PMC5713628.</p></li>
<li><p>Laura Rayhel, B.A., Copper Aitken-Palmer, D.V.M., Ph.D.,
Priscilla Joyner, B.Sc., B.V.M.S., Carolyn Cray, Ph.D., <strong>César
Andrés Lizárraga, B.A.</strong>, Betty Ackerman, M.T. (A.S.C.P.), and
Chris Crowe, B.S. <strong>Hematology and biochemistry in captive
white-naped cranes (<em>Grus VIPIO</em>)</strong>. <em>Journal of Zoo
and Wildlife Medicine 46(4):747-754.
http://dx.doi.org/10.1638/2015-0027.1 (2015)</em></p></li>
</ul>
<h2 id="professional-presentations-posters">Professional Presentations
(Posters)</h2>
<ul>
<li><p>Skyler Mitchell, Stuart Marshall, Stephanie Turnipseed, Luke
Burnham, <strong>César Lizárraga</strong>, Jared Streich, Rob Alba, and
Todd C. Mockler (2015) “Effect of drought treatments on transpiration
rate and stomatal density in <em>Brachypodium distachyon</em>.” Donald
Danforth Plant Science Center and Missouri Botanical Gardens Joint Fall
Symposium, St. Louis, MO, October 2015</p></li>
<li><p><strong>César A. Lizárraga</strong>, Henry D. Priest, Noah
Fahlgren, Rob Alba, and Todd C. Mockler. Bioinformatics Pipelines for
Purple False Brome (Brachypodium distachyon) Donald Danforth Plant
Science Center, St. Louis, MO. (2015)</p></li>
<li><p><strong>Cesar Lizarraga</strong>, Stuart Marshall, Bradley Flynn,
Nadia Shakoor and Todd C. Mockler. PhenoPiSight: Fixed Camera
Greenhouse-based Phenotyping Platform. (2016)</p></li>
</ul>
</body>
</html>
