<?xml version="1.0" encoding="UTF-8" ?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
	"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">

<html xmlns="http://www.w3.org/1999/xhtml">

<head>
<title>Markdown_Resume.html</title>

</head>

<body>

<h1>César A. Lizárraga</h1>

<p>e-mail: <a href="&#x6D;&#97;&#105;&#108;&#116;&#x6F;:&#x63;&#97;&#x6C;&#x69;&#122;&#97;&#x72;&#114;&#64;&#103;&#x6D;&#x61;&#x69;&#108;&#46;c&#111;&#109;">&#x63;&#97;&#x6C;&#x69;&#122;&#97;&#x72;&#114;&#64;&#103;&#x6D;&#x61;&#x69;&#108;&#46;c&#111;&#109;</a> <br />
github: <a href="http://github.com/calizarr">http://github.com/calizarr</a> <br />
LinkedIn: <a href="http://www.linkedin.com/in/calizarr">http://linkedin.com/in/calizarr</a></p>

<h2>Work Experience</h2>

<ul>
<li><p><strong><a href="https://www.cibotechnologies.com/">CiBO Technologies</a> (St. Louis, MO): July 2017 - Present</strong> <br />
<em>Engagement Engineering Lead (Project Lead / Software Engineer)</em></p>

<ul>
<li>Lead a small team (4-5 engineers) in custom appliation development for a strategic partnership client</li>
<li>Participated in planning multiple stages of the project at short and medium time phases</li>
<li>Regularly interfaced with the client team to ensure needs and desires were met</li>
<li>Using a high level concept, designed and architected an industry leading set of tools that allowed the client to reduce their research time from weeks and months to days</li>
<li>Helped prototype, develop, and deploy several services, APIs, and API clients in kubernetes</li>
</ul></li>
<li><p><strong><a href="http://www.nanaya.co">Nanaya</a> (St. Louis, MO): July 2014 - Present</strong> <br />
<em>Co-founder/ Infrastructure &amp; Sofware Engineer / Statistician</em></p>

<ul>
<li>Develop, design, and test algorithm(s) from algorithm</li>
<li>Verify that proper statistical procedures are being used within the algorithm(s)</li>
<li>Facilitate communication between developers and R&amp;D Team members</li>
<li>Manage and update infrastructure depending on needs of team</li>
</ul></li>
<li><p><strong><a href="https://www.danforthcenter.org/">Donald Danforth Plant Science Center</a> (St. Louis, MO)</strong></p>

<ul>
<li><strong>October 2016 - July 2017</strong> <br />
<em>Senior Computational Scientist</em>
<ul>
<li>Provided computational interface for research groups</li>
<li>Trained several lab technicians in computational infrastructure use</li>
<li>Developed prototype of Arduino &amp; Raspberry Pi controlled field canopy sensor system: <a href="https://www.agrelaeco.com/">PheNode</a></li>
<li><a href="#phenopisight">PhenoPiSight: A Fixed Camera Greenhouse-based Phenotyping platform</a></li>
</ul></li>
<li><strong>December 2014 - October 2016</strong> <br />
<em>Laboratory Technician (Bioinformatics/Statistics) in Mockler Lab</em> <br />
<ul>
<li>Developed, documented, maintained, and tested pipelines for analyzing high throughput sequencing and imaging data</li>
<li>Analytics, Data Management, and Development for the <a href="http://genomicscience.energy.gov/research/DOEUSDA/abstracts/2014mockler_abstract.shtml">Brachypodium ENCODE Project</a> and the <a href="https://missouriepscor.org/">EPSCoR Project</a></li>
<li>Coordinate with Bioinformatics Core Director for computational resource management</li>
<li>Assist with data analysis whenever requested</li>
</ul></li>
</ul></li>
<li><p><strong><a href="http://www.bensonhillbio.com/">Benson Hill Biosystems</a> (St. Louis, MO): December 2014 - September 2015</strong> <br />
<em>Independent Contractor (Bioinformatics/Statistics)</em></p>

<ul>
<li>Developed a high throughput imaging analysis pipeline for a custom in-house imaging system &amp; an RNA-seq pipeline to discover novel transcripts</li>
<li>Provided statistical consultation when necessary</li>
</ul></li>
</ul>

<h2>Project Experience</h2>

<h4>PhenoPiSight: Fixed Camera Greenhouse-based Phenotyping Platform<a name="phenopisight"></a></h4>

<ul>
<li>Using Ansible, automated image capture and transfer of images from 180 Raspberry Pis on a gantry above the greenhouse</li>
<li>Developed pipeline to take captured images and make dense 3d pointcloud reconstructions (+/- 0.5cm accuracy)</li>
<li>Trained lab technicians to find phenotypes in 3d reconstructions and compare to ground-based greenhouse measurements</li>
<li><a href="https://traitcapture.org/pointclouds/by-id/586a428ef7f5667846b1f8a0">Example of the 3d reconstruction</a></li>
</ul>

<h2>Education</h2>

<p><strong>B.A. Mathematics, 2008</strong> <br />
<em>Probability and Statistics</em> <br />
Department of Mathematics, Washington University in St. Louis  </p>

<h2>Engineering Skills</h2>

<ul>
<li>In order of familiarity:
<ul>
<li>Languages: <em>Scala, Python, R, Bash, Java</em></li>
<li>Infrastructure &amp; DevOps: <em>Kubernetes, Helm, Argo Workflow Engine, Ansible</em></li>
<li>Operating systems and GNU/Linux: <em>grep, AWK, sed</em></li>
</ul></li>
</ul>

<h2>Languages</h2>

<ul>
<li>Fluent: <em>English</em>, <em>Spanish</em></li>
<li>Intermediate: <em>Italian</em></li>
</ul>

<h2>Publications</h2>

<ul>
<li>Erica Agnew, Adam Bray, Eric Floro, Nate Ellis, John Gierer, <strong>César Lizárraga</strong>, Darren O'Brien, Madeline Wiechert, Todd C. Mockler, Nadia Shakoor, Christopher N. Topp. Whole‐Plant Manual and Image‐Based Phenotyping in Controlled Environments. <em>Plant Biology 2(1):1-21. https://doi.org/10.1002/cppb.20044 (2017)</em></li>
<li>He Huang, Malia A. Gehan, Sarah E. Huss, Sophie Alvarez, <strong>Cesar Lizarraga</strong>, Ellen L. Gruebbling, John Gierer, Michael J. Naldrett, Rebecca K. Bindbeutel, Bradley S. Evans, Todd C. Mockler, Dmitri A. Nusinow. Cross‐species complementation reveals conserved functions for EARLY FLOWERING 3 between monocots and dicots. <em>Plant Direct 1:4 https://doi.org/10.1002/pld3.18 (2017)</em></li>
<li>Gehan MA, Fahlgren N, Abbasi A, Berry JC, Callen ST, Chavez L, Doust AN, Feldman MJ, Gilbert KB, Hodge JG, Hoyer JS, Lin A, Liu S, <strong>Lizárraga C</strong>, Lorence A, Miller M, Platon E, Tessman M, Sax T. 2017. PlantCV v2: Image analysis software for high-throughput plant phenotyping. PeerJ 5:e4088 https://doi.org/10.7717/peerj.4088</li>
<li>Laura Rayhel, B.A., Copper Aitken-Palmer, D.V.M., Ph.D., Priscilla Joyner, B.Sc., B.V.M.S., Carolyn Cray, Ph.D., <strong>César Andrés Lizárraga, B.A.</strong>, Betty Ackerman, M.T. (A.S.C.P.), and Chris Crowe, B.S. Hematology and biochemistry in captive white-naped cranes (<em>Grus VIPIO</em>). <em>Journal of Zoo and Wildlife Medicine 46(4):747-754. http://dx.doi.org/10.1638/2015-0027.1 (2015)</em></li>
</ul>

<h2>Professional Presentations (Posters)</h2>

<ul>
<li>Skyler Mitchell, Stuart Marshall, Stephanie Turnipseed, Luke Burnham, <strong>César Lizárraga</strong>, Jared Streich, Rob Alba, and Todd C. Mockler (2015) "Effect of drought treatments on transpiration rate and stomatal density in <em>Brachypodium distachyon</em>." Donald Danforth Plant Science Center and Missouri Botanical Gardens Joint Fall Symposium, St. Louis, MO, October 2015</li>
<li><strong>César A. Lizárraga</strong>, Henry D. Priest, Noah Fahlgren, Rob Alba, and Todd C. Mockler. Bioinformatics Pipelines for Purple False Brome (Brachypodium distachyon) Donald Danforth Plant Science Center, St. Louis, MO. (2015)</li>
<li><strong>Cesar Lizarraga</strong>, Stuart Marshall, Bradley Flynn, Nadia Shakoor1 and Todd C. Mockler. PhenoPiSight: Fixed Camera Greenhouse-based Phenotyping Platform. (2016)</li>
</ul>

</body>
</html>
