# César A. Lizárraga #
e-mail: <calizarr@gmail.com>  
github: [http://github.com/calizarr] [git]  
LinkedIn: [http://linkedin.com/in/calizarr] [linkedin]

### Education ###
**B.A. Mathematics, 2008**  
*Probability and Statistics*  
Department of Mathematics, Washington University in St. Louis

### Work Experience ###

  * **December 2014 - Present: [Donald Danforth Plant Science Center][ddpsc] (St. Louis, MO)**
  * *Laboratory Technician (Bioinformatics/Statistics)*
      * Develop and document pipelines for analyzing high throughput
        sequencing and imaging data.
      * Analytics and Data Management for the [Brachypodium ENCODE Project][encode].
  * **July 2014 - Present: [Nanaya][nanaya] (St. Louis, MO)**
  * *Statistician/Back-end developer*
      * Develop, design, and test algorithm(s) from prototype.
      * Verify that proper statistical procedures are being used
        within prototype algorithm(s).
      * Facilitate communication between developers and R&D Team members.
      * Manage and update server depending on needs of team.
  * **December 2014 - September 2015: [Benson Hill Biosystems][BHB] (St. Louis, MO)**
  * *Independent Contractor (Bioinformatics/Statistics)*
      * Developed high throughput imaging analysis pipeline for
        a custom in-house imaging system.
      * Developed an RNA-seq pipeline to discover novel transcripts.
      * Provided statistical consultation when necessary.

### Project Experience ###

  * **Brachypodium ENCODE Project**
      * Develop, maintain, and test a SNP calling pipeline for NGS paired-end read data
        from Illumina sequencing platform.
          * [Developed in Python and Bash wrapper script: PyPipe] [pypipe]
      * Develop, maintain, and test a ChIP-seq pipeline for NGS data from Illumina
        sequencing platform.
          * [Developed in Python: PyChIP] [pychip]
      * Developed high throughput phenotyping pipeline using PlantCV
        and R for imaging analysis for internal use with LemnaTec HTP
        facility.
      * Mapped forward previous structural *Brachypodium distachyon*
        annotations to new genome assembly using MAKER. Mapped
        functional (PFAM, PHOBIUS, and TIGRFAM.) domains with
        InterProScan to the annotations and converted locations to
        genomic coordinates. Structural and functional annotations
        were placed as tracks in a genome browser.
  * **Benson Hill Biosystems**
      * Developed high throughput phenotyping pipeline using HTPheno,
        R, and Python for a custom in-house imaging system.
      * Developed an RNA-seq pipeline to discover novel transcripts
        using structural annotations, a new genome assembly, and set
        intersections. 

### Bioinformatics Skills ###
  * NGS data analysis: *RNA-seq, ATAC-seq, ChIP-seq, variant calling*
  * Biostatistics: *Regression, multiple test correction, modelling,
    heritability (broad-sense)*
  * Clustering analysis data: *UPGMA, K-means, Agglomerative
    hierarchical clustering*
  * Databases and repositories: *Ensembl, Phytozome, NCBI Gene
    Expression Omnibus*
  * Libraries and tools: *SAMtools, Bioconductor, BioPython, SICER,
    MACS2, BWA, Blast, Trimmomatic, Bowtie, STAR, GATK2, PicardTools,
    Variant Effect Predictor, HTPheno, PlantCV, InterProScan, Tophat,
    Cufflinks*

### Computer Skills ###
  * Languages: *Python, R, Perl, Java, MATLAB*
  * Operating systems and GNU/Linux: *Shell script (bash), AWK, sed,
  grep, Emacs, UNIX*

### Languages ###
  * Spanish: *Fluent*
  * Italian: *Intermediate*

### Publications ###
  * Laura Rayhel, B.A., Copper Aitken-Palmer, D.V.M., Ph.D.,
    Priscilla Joyner, B.Sc., B.V.M.S., Carolyn Cray, Ph.D., **César
    Andrés Lizárraga, B.A.**, Betty Ackerman, M.T.(A.S.C.P.), and Chris
    Crowe, B.S. Hematology and biochemistry in captive white-naped
    crane (*Grus vipio*). *Journal of Zoo and Wildlife Medicine (in press)*

### Posters ###
  * Skyler Mitchell, Stuart Marshall, Stephanie Turnipseed, Luke
    Burnham, **César Lizárraga**, Jared Streich, Rob Alba, and Todd
    C. Mockler (2015) "Effect of drought treatments on transpiration
    rate and stomatal density in *Brachypodium distachyon*." Donald
    Danforth Plant Science Center and Missouri Botanical Gardens Joint
    Fall Symposium, St. Louis, MO, October 2015.
  

[git]: http://github.com/calizarr
[linkedin]: http://www.linkedin.com/in/calizarr
[ddpsc]: https://www.danforthcenter.org/
[encode]: http://genomicscience.energy.gov/research/DOEUSDA/abstracts/2014mockler_abstract.shtml
[BHB]: http://www.bensonhillbio.com/
[pypipe]: https://github.com/calizarr/PyPipeline/
[pychip]: https://github.com/calizarr/PyChIP-seq
[nanaya]: http://www.nanaya.co
