<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <meta http-equiv="Content-Style-Type" content="text/css" />
  <meta name="generator" content="pandoc" />
  <title></title>
  <style type="text/css">code{white-space: pre;}</style>
  <link rel="stylesheet" href="buttondown.css" type="text/css" />
</head>
<body>
<h1 id="c&#233;sar-a.-liz&#225;rraga">C&#233;sar A. Liz&#225;rraga</h1>
<p>e-mail: <script type="text/javascript">
<!--
h='&#x67;&#x6d;&#x61;&#x69;&#108;&#46;&#x63;&#x6f;&#x6d;';a='&#64;';n='&#x63;&#x61;&#108;&#x69;&#122;&#x61;&#114;&#114;';e=n+a+h;
document.write('<a h'+'ref'+'="ma'+'ilto'+':'+e+'">'+e+'<\/'+'a'+'>');
// -->
</script><noscript>&#x63;&#x61;&#108;&#x69;&#122;&#x61;&#114;&#114;&#32;&#x61;&#116;&#32;&#x67;&#x6d;&#x61;&#x69;&#108;&#32;&#100;&#x6f;&#116;&#32;&#x63;&#x6f;&#x6d;</noscript><br />github: <a href="http://github.com/calizarr">http://github.com/calizarr</a><br />LinkedIn: <a href="http://www.linkedin.com/in/calizarr">http://linkedin.com/in/calizarr</a></p>
<h2 id="work-experience">Work Experience</h2>
<ul>
<li><strong>December 2014 - Present: <a href="https://www.danforthcenter.org/">Donald Danforth Plant Science Center</a> (St. Louis, MO)</strong><br /> <em>Laboratory Technician (Bioinformatics/Statistics) in Mockler Lab</em>
<ul>
<li>Develop and document pipelines for analyzing high throughput sequencing and imaging data</li>
<li>Analytics and Data Management for the <a href="http://genomicscience.energy.gov/research/DOEUSDA/abstracts/2014mockler_abstract.shtml">Brachypodium ENCODE Project</a></li>
<li>Analytics and Data Management for the <a href="https://missouriepscor.org/">EPSCoR Project</a></li>
<li>Assist with data analysis whenever requested</li>
</ul></li>
<li><strong>July 2014 - Present: <a href="http://www.nanaya.co">Nanaya</a> (St. Louis, MO)</strong><br /> <em>Statistician/Back-end developer</em>
<ul>
<li>Develop, design, and test algorithm(s) from prototype</li>
<li>Verify that proper statistical procedures are being used within prototype algorithm(s)</li>
<li>Facilitate communication between developers and R&amp;D Team members</li>
<li>Manage and update server depending on needs of team</li>
</ul></li>
<li><strong>December 2014 - September 2015: <a href="http://www.bensonhillbio.com/">Benson Hill Biosystems</a> (St. Louis, MO)</strong><br /> <em>Independent Contractor (Bioinformatics/Statistics)</em>
<ul>
<li>Developed high throughput imaging analysis pipeline for a custom in-house imaging system</li>
<li>Developed an RNA-seq pipeline to discover novel transcripts</li>
<li>Provided statistical consultation when necessary</li>
</ul></li>
</ul>
<h2 id="education">Education</h2>
<p><strong>B.A. Mathematics, 2008</strong><br /><em>Probability and Statistics</em><br />Department of Mathematics, Washington University in St. Louis</p>
<h2 id="project-experience">Project Experience</h2>
<ul>
<li><strong>General Lab Projects</strong>
<ul>
<li>Develop, maintain, and test an RNA-seq pipeline for NGS data from Illumina sequencing platform</li>
<li>Develop, maintain, and test a SNP calling pipeline for NGS paired-end read data from Illumina sequencing platform
<ul>
<li><a href="https://github.com/calizarr/PyPipeline/">Developed in Python and Bash wrapper script: PyPipe</a></li>
</ul></li>
<li>Develop, maintain, and test a ChIP-seq pipeline for NGS data from Illumina sequencing platform
<ul>
<li><a href="https://github.com/calizarr/PyChIP-seq">Developed in Python: PyChIP</a></li>
</ul></li>
<li>Developed high throughput phenotyping pipeline using PlantCV and R for imaging analysis for internal use with LemnaTec high-throughput facility</li>
</ul></li>
<li><strong>Brachypodium ENCODE Project</strong>
<ul>
<li>Mapped forward previous structural <em>Brachypodium distachyon</em> annotations to new genome assembly using MAKER</li>
<li>Mapped functional domains with InterProScan to the annotations and converted locations to genomic coordinates</li>
<li>Structural and functional annotations were placed as tracks in a genome browser</li>
</ul></li>
<li><strong>PheNode: Field canopy sensor system</strong>
<ul>
<li>Develop code for Arduino to interface with sensors</li>
<li>Develop code to control Arduino from Raspberry Pi, read, and save results</li>
<li>Assess power requirements and setup batteries, relays, etc.</li>
</ul></li>
<li><strong>PhenoPiSight: Fixed Camera Greenhouse-based Phenotyping Platform</strong>
<ul>
<li>Automate image capture and transfer of images from 180 Raspberry Pis on a gantry above the greenhouse</li>
<li>Develop pipeline to take captured images and make dense 3d reconstructions</li>
<li>Manually find phenotypes in 3d reconstructions and compare to ground-based greenhouse measurements</li>
<li><a href="https://github.com/calizarr/EPSCoR_Bramble_GH9C">Developed in Python and Bash: PhenoPiSight</a></li>
</ul></li>
<li><strong>Benson Hill Biosystems</strong>
<ul>
<li>Developed high throughput phenotyping pipeline using HTPheno, R, and Python for a custom in-house imaging system</li>
<li>Developed an RNA-seq pipeline to discover novel transcripts using structural annotations, a new genome assembly, and set intersections.</li>
</ul></li>
</ul>
<h2 id="bioinformatics-skills">Bioinformatics Skills</h2>
<ul>
<li>NGS data analysis: <em>RNA-seq, ATAC-seq, ChIP-seq, variant calling</em></li>
<li>Biostatistics: <em>Regression, multiple test correction, modelling, heritability (broad-sense)</em></li>
<li>Clustering analysis data: <em>UPGMA, K-means, Agglomerative, hierarchical clustering</em></li>
<li>Databases and repositories: <em>Ensembl, Phytozome, NCBI Gene Expression Omnibus</em></li>
</ul>
<h2 id="computer-skills">Computer Skills</h2>
<ul>
<li>Languages: <em>Python, R, Bash, Perl, Java, MATLAB</em></li>
<li>Operating systems and GNU/Linux: <em>AWK, sed, grep, Emacs, UNIX</em></li>
</ul>
<h2 id="languages">Languages</h2>
<ul>
<li>Spanish: <em>Fluent</em></li>
<li>Italian: <em>Intermediate</em></li>
</ul>
<h2 id="publications">Publications</h2>
<ul>
<li>Laura Rayhel, B.A., Copper Aitken-Palmer, D.V.M., Ph.D., Priscilla Joyner, B.Sc., B.V.M.S., Carolyn Cray, Ph.D., <strong>C&#233;sar Andr&#233;s Liz&#225;rraga, B.A.</strong>, Betty Ackerman, M.T. (A.S.C.P.), and Chris Crowe, B.S. Hematology and biochemistry in captive white-naped cranes (<em>Grus VIPIO</em>). <em>Journal of Zoo and Wildlife Medicine 46(4):747-754. http://dx.doi.org/10.1638/2015-0027.1 (2015)</em></li>
</ul>
<h2 id="professional-presentations">Professional Presentations</h2>
<ul>
<li>Skyler Mitchell, Stuart Marshall, Stephanie Turnipseed, Luke Burnham, <strong>C&#233;sar Liz&#225;rraga</strong>, Jared Streich, Rob Alba, and Todd C. Mockler (2015) &quot;Effect of drought treatments on transpiration rate and stomatal density in <em>Brachypodium distachyon</em>.&quot; Donald Danforth Plant Science Center and Missouri Botanical Gardens Joint Fall Symposium, St. Louis, MO, October 2015</li>
<li><strong>C&#233;sar A. Liz&#225;rraga</strong>, Henry D. Priest, Noah Fahlgren, Rob Alba, and Todd C. Mockler. Bioinformatics Pipelines for Purple False Brome (Brachypodium distachyon) Donald Danforth Plant Science Center, St. Louis, MO. (2015)</li>
<li><strong>Cesar Lizarraga</strong>, Stuart Marshall, Bradley Flynn, Nadia Shakoor1 and Todd C. Mockler. PhenoPiSight: Fixed Camera Greenhouse-based Phenotyping Platform. (2016)</li>
</ul>
</body>
</html>
