César A. Lizárraga
==================

e-mail: <calizarr@gmail.com>  
github: [http://github.com/calizarr] [git]  
LinkedIn: [http://linkedin.com/in/calizarr] [linkedin]

Work Experience
---------------

* **December 2014 - Present: [Donald Danforth Plant Science Center][ddpsc] (St. Louis, MO)**  
    *Laboratory Technician (Bioinformatics/Statistics) in Mockler Lab*
    * Develop and document pipelines for analyzing high throughput sequencing and imaging data
    * Analytics and Data Management for the [Brachypodium ENCODE Project][encode]
    * Analytics and Data Management for the [EPSCoR Project][EPSCoR]
    * Assist with data analysis whenever requested
* **July 2014 - Present: [Nanaya][nanaya] (St. Louis, MO)**  
    *Statistician/Back-end developer*
    * Develop, design, and test algorithm(s) from prototype
	* Verify that proper statistical procedures are being used within prototype algorithm(s)
	* Facilitate communication between developers and R&D Team members
	* Manage and update server depending on needs of team
* **December 2014 - September 2015: [Benson Hill Biosystems][BHB] (St. Louis, MO)**  
    *Independent Contractor (Bioinformatics/Statistics)*
	* Developed high throughput imaging analysis pipeline for a custom in-house imaging system
	* Developed an RNA-seq pipeline to discover novel transcripts
	* Provided statistical consultation when necessary

Education
---------

**B.A. Mathematics, 2008**  
*Probability and Statistics*  
Department of Mathematics, Washington University in St. Louis

Project Experience
------------------

* **General Lab Projects**
    * Develop, maintain, and test an RNA-seq pipeline for NGS data from Illumina sequencing platform
    * Develop, maintain, and test a SNP calling pipeline for NGS paired-end read data from Illumina sequencing platform
        - [Developed in Python and Bash wrapper script: PyPipe] [pypipe]
    * Develop, maintain, and test a ChIP-seq pipeline for NGS data from Illumina sequencing platform
        - [Developed in Python: PyChIP] [pychip]
    * Developed high throughput phenotyping pipeline using PlantCV and R for imaging analysis for internal use with LemnaTec high-throughput facility
* **Brachypodium ENCODE Project**
    * Mapped forward previous structural *Brachypodium distachyon* annotations to new genome assembly using MAKER
    * Mapped functional domains with InterProScan to the annotations and converted locations to genomic coordinates
    * Structural and functional annotations were placed as tracks in a genome browser
* **PheNode: Field canopy sensor system**
    * Develop code for Arduino to interface with sensors
    * Develop code to control Arduino from Raspberry Pi, read, and save results
    * Assess power requirements and setup batteries, relays, etc.
* **PhenoPiSight: Fixed Camera Greenhouse-based Phenotyping Platform**
    * Automate image capture and transfer of images from 180 Raspberry Pis on a gantry above the greenhouse
    * Develop pipeline to take captured images and make dense 3d reconstructions
    * Manually find phenotypes in 3d reconstructions and compare to ground-based greenhouse measurements
    * [Developed in Python and Bash: PhenoPiSight] [phenopisight]
* **Benson Hill Biosystems**
    * Developed high throughput phenotyping pipeline using HTPheno, R, and Python for a custom in-house imaging system
    * Developed an RNA-seq pipeline to discover novel transcripts using structural annotations, a new genome assembly, and set intersections. 

Bioinformatics Skills
---------------------

* NGS data analysis: *RNA-seq, ATAC-seq, ChIP-seq, variant calling*
* Biostatistics: *Regression, multiple test correction, modelling, heritability (broad-sense)*
* Clustering/Dimensionality reduction: *UPGMA, K-means, Agglomerative, hierarchical clustering, PCA*

Computer Skills
---------------

* Languages: *Python, R, Bash, Perl, Java, MATLAB*
* Operating systems and GNU/Linux: *AWK, sed, grep, Emacs, UNIX*

Languages
---------

* Spanish: *Fluent*
* Italian: *Intermediate*

Publications
------------

* Laura Rayhel, B.A., Copper Aitken-Palmer, D.V.M., Ph.D., Priscilla Joyner, B.Sc., B.V.M.S., Carolyn Cray, Ph.D., **César Andrés Lizárraga, B.A.**, Betty Ackerman, M.T. (A.S.C.P.), and Chris Crowe, B.S. Hematology and biochemistry in captive white-naped cranes (*Grus VIPIO*). *Journal of Zoo and Wildlife Medicine 46(4):747-754. http://dx.doi.org/10.1638/2015-0027.1 (2015)*

Professional Presentations (Posters)
--------------------------

* Skyler Mitchell, Stuart Marshall, Stephanie Turnipseed, Luke Burnham, **César Lizárraga**, Jared Streich, Rob Alba, and Todd C. Mockler (2015) "Effect of drought treatments on transpiration rate and stomatal density in *Brachypodium distachyon*." Donald Danforth Plant Science Center and Missouri Botanical Gardens Joint Fall Symposium, St. Louis, MO, October 2015
* **César A. Lizárraga**, Henry D. Priest, Noah Fahlgren, Rob Alba, and Todd C. Mockler. Bioinformatics Pipelines for Purple False Brome (Brachypodium distachyon) Donald Danforth Plant Science Center, St. Louis, MO. (2015)
* **Cesar Lizarraga**, Stuart Marshall, Bradley Flynn, Nadia Shakoor1 and Todd C. Mockler. PhenoPiSight: Fixed Camera Greenhouse-based Phenotyping Platform. (2016)

[Git]: http://github.com/calizarr
[linkedin]: http://www.linkedin.com/in/calizarr
[ddpsc]: https://www.danforthcenter.org/
[encode]: http://genomicscience.energy.gov/research/DOEUSDA/abstracts/2014mockler_abstract.shtml
[BHB]: http://www.bensonhillbio.com/
[pypipe]: https://github.com/calizarr/PyPipeline/
[pychip]: https://github.com/calizarr/PyChIP-seq
[nanaya]: http://www.nanaya.co
[EPSCoR]: https://missouriepscor.org/
[phenopisight]: https://github.com/calizarr/EPSCoR_Bramble_GH9C
