# Cesar A. Lizarraga#
e-mail: <calizarr@gmail.com>  
[github: http://github.com/calizarr] [git]  
[LinkedIn: http://linkedin.com/in/calizarr] [linkedin]

### Education###
**B.A. Mathematics, 2008**  
*Probability and Statistics*  
Department of Mathematics, Washington University in St. Louis (US)

### Experience ###

#### Work Experience ####

  * **December 2014 -- present: [Donald Danforth Plant Science Center][ddpsc]**
      * St. Louis, MO
      * *Lab Technician (Bioinformatics/Statistics)*
      * Develop and document pipelines for analyzing high throughput
      sequencing data.
          * *ChIP-seq analysis pipeline, SNP calling pipeline, and HTP
            imaging analsysis pipeline for LemnaTec HTP facility* 
      * Analyse and manage data for the [Brachypodium ENCODE Project][encode].
  * **December 2014 -- September 2015: [Benson Hill Biosystems][BHB]**
      * St. Louis, MO
      * *Independent Contractor (Bioinformatics/Statistics)*
      * Developed high throughput imaging analysis pipeline for
        homegrown imaging system.
      * Provided statistical consultation when necessary.

#### Project Experience ####
  * **Brachypodium Encode Project**
      * Developed a SNP calling pipeline for NGS paired-end read data
        from Illumina sequencing machines.
          * [PyPipe] [pypipe]
      * Developed a ChIP-seq pipeline for NGS data from Illumina
        sequencing machines using controls or random background as
        control.
          * [PyChIP] [pychip]

### Bioinformatics Skills###
  * NGS data analysis: *RNA-seq, ChIP-seq, variant calling*
  * Biostatistics: *regression, multiple test correction, modelling*
  * Predictors and clustering analysis of data: *UPGMA,
    K-means, Agglomerative hierarchical clustering*
  * Databases and repositories: *Ensembl, Phytozome, NCBI Gene
    Expression Omnibus*
  * Libraries and tools: *SAMtools, Bioconductor, BioPython, SICER,
    MACS2, BWA, Blast, Trimmomatic, Bowtie, STAR, GATK2, PicardTools,
    Variant Effect Predictor, HTPheno, PlantCV, InterProScan ...*

### Computer Skills ###
  * Languages: *Python, R, Perl, Java, ...*
  * Operating systems and GNU/Linux: *Shell script (bash), AWK, sed,
  grep, ...*

[git]: http://github.com/calizarr
[linkedin]: http://www.linkedin.com/in/calizarr
[ddpsc]: https://www.danforthcenter.org/
[encode]: http://genomicscience.energy.gov/research/DOEUSDA/abstracts/2014mockler_abstract.shtml
[BHB]: http://www.bensonhillbio.com/
[pypipe]: https://github.com/calizarr/PyPipeline/blob/master/README.md
[pychip]: https://github.com/calizarr/PyChIP-seq
